docs/portal_config.md

```diff
@@ -78,6 +79,8 @@ Below is an example, with inline comments describing what each JSON block config
           "label": "Run analysis"
         }
       ],
+      "homepageChartNodesExcludeFiles": true, // optional; specifies whether to exclude File nodes from the homepage chart. Defaults to false
```

Comment:
i think it's not in the right place, shouldn't it be under `components.index`?

Reply:
the format of this file is a bit confusing, but this one is actually under `components.index` now